Documentation/manual.txt

```diff
@@ -122,12 +122,10 @@ Options:
 	Relevant for dtb and asm output only.
```

Comment:
Not shortening the description isn't a virtue of itself.  However, the information you've added is potentially useful, so the patch is good.